opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java

```diff
@@ -38,6 +39,11 @@ public ScopeManager scopeManager() {
     @Override
     public <C> SpanContext extract(Format<C> format, C carrier) { return NoopSpanBuilderImpl.INSTANCE; }
 
+    @Override
+    public Span activeSpan() {
+        return NoopSpan.INSTANCE;
```

Comment:
IIRC in `0.30.0` this returned `null` which could cause NPEs

It's probably better to return something.

Reply:
Agree - although it might also be good to have some way to determine if there is an active span, in case an instrumentation wants to avoid doing unnecessary work. Possibly an additional method on the `ScopeManager.hasActiveSpan()`?

Reply:
good point. 

> Possibly an additional method on the ScopeManager.hasActiveSpan()?

This seems like a duplicate of `Span activeSpan()`.

Reply:
It's related to https://github.com/opentracing/opentracing-java/issues/134. Maybe it's better to return null and in future add a convenient Optional-like method.

Reply:
Or we could make noop to use span manager and return what is actually there.